provider/resources.go

```diff
@@ -918,6 +920,7 @@ func Provider() tfbridge.ProviderInfo {
 				},
 			},
 			"google_dataproc_autoscaling_policy": {Tok: gcpResource(gcpDataProc, "AutoscalingPolicy")},
+			"google_dataproc_metastore_service":  {Tok: gcpResource(gcpDataProc, "MetaStoreService")},
```

Comment:
It seems to be "Metastore" in the docs https://cloud.google.com/dataproc-metastore/docs

Reply:
Fixed!